com.unity.cinemachine/Documentation~/FreeLookCameras.md

```diff
@@ -8,9 +8,9 @@ When you choose **Cinemachine > FreeLook Camera** from the GameObject menu, it w
 
 ![Default Free Look Inspector](images/DefaultFreeLookInspector.png)
 
-The [Orbital Follow](CinemachineOrbitalFollow.md) provides the orbiting functionality, and the [Rotation Composer](CinemachineRotationComposer.md) ensures that the camera always looks at the target, from whatever position it is.  User input is provided by the [Input Axis Controller](CinemachineInputAxisController.md) component. The settings there will allow you to map the camera angle to whatever input controls you like, and to tune their responsiveness.
+The [Orbital Follow](CinemachineOrbitalFollow.md) provides the orbiting position functionality, and the [Rotation Composer](CinemachineRotationComposer.md) ensures that the camera always looks at the target, from whatever position it is.  User input is provided by the [Input Axis Controller](CinemachineInputAxisController.md) component. The settings there will allow you to map the camera angle to whatever input controls you like, and to tune their responsiveness.
 
-The [Cinemachine Free Look Modifier](CinemachineFreeLookModifier.md) behavior's job is to allow you to tweak some settings as a function of the vertical veiwing angle of the camera. For example, you can have different lens settings depending on whether you look from the top or from the bottom. A number of different possibilities are provided with that component, and of course it is always possible to craft your own and add them.
+The [Cinemachine Free Look Modifier](CinemachineFreeLookModifier.md) behavior's job is to allow you to tweak some settings as a function of the vertical veiwing angle of the camera. For example, you can have different lens settings depending on whether you look from the top or from the bottom. A number of different possibilities are provided with that component, and of course it is always possible to craft your own and add them.  Cinemachine Free Look Modifier is an optional component, and you can remove it if you don't need what it provides.
 
 An alternative version of a FreeLook can be made by changing the Position Control behavior to [Position Composer](CinemachinePositionComposer.md) and the Rotation Control behavior to [Pan Tilt](CinemachinePanTilt.md), as follows:
```

Comment:
We will need to expand on this at some point, to explain the differences between these models and give some clues about why you would want one rather than the other.

Reply:
Different file, but same comment as above.
And yes, totally! The _why_ is my quest :)